src/main/java/io/kokuwa/maven/helm/AbstractChartDirectoryMojo.java

```diff
@@ -0,0 +1,67 @@
+package io.kokuwa.maven.helm;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileVisitOption;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.codehaus.plexus.util.FileUtils;
+import org.codehaus.plexus.util.MatchPatterns;
+
+import lombok.Setter;
+
+/** Base class for any mojo requiring a chart directory parameter */
+@Setter
+public abstract class AbstractChartDirectoryMojo extends AbstractHelmMojo {
```

Comment:
There are MOJO's that do not actually use the chartDirectory parameter including the MOJO that I have extended in another project. If I didnt, it would have to be set even though it is never used.